cmd/build/v2/environment.go

```diff
@@ -55,5 +55,15 @@ func (bc *OktetoBuilder) SetServiceEnvVars(service, reference string) {
 	os.Setenv(tagKey, tag)
 	bc.lock.Unlock()
 
+	sha := tag
+	if strings.HasPrefix(sha, "sha256:") {
+		sha = fmt.Sprintf("okteto@%s", sha)
```

Comment:
Should we use [this constant](https://github.com/okteto/okteto/blob/master/pkg/model/const.go#L329) instead of harcoding the `okteto` value? 

If I'm not wrong, we have to use the name of the tag that was used when building the image, right? Or can we set any value there?

Reply:
You can set any arbitrary string. what's not supported is this:
```
my-image:sha256xxx
```
You need to do either  `my-image@sha256xxx` or `my-image:anything@sha256xxx`